src/data/chunk_list.cc

```diff
@@ -171,7 +171,7 @@ ChunkList::release(ChunkHandle* handle, release_flags flags) {
   if (!handle->is_valid())
     throw internal_error("ChunkList::release(...) received an invalid handle.");
 
-  if (handle->object() < &*begin() || handle->object() >= &*end())
+  if (handle->object() < &front() || handle->object() >= &back())
```

Comment:
End is off by one, should be `>`.

However I don't like this change as begin/end are more natural for iterators/position comparisons.

Reply:
No, I mean it is more natural to use begin/end when dealing with comparison of position.

Reply:
Removed.